tests/js/sample/bugs/227/minified.js

```diff
@@ -1,3 +1,3 @@
-var escapeSelector=$.ui.escapeSelector=(function(){var selectorEscape=/([!"#$%&'()*+,./:;<=>?@[\]^`{|}~])/g;return function(selector){return selector.replace(selectorEscape,"\\$1")}})();var labels=$.fn.labels=function(){var ancestor,selector,id,labels,ancestors;if(this[0].labels&&this[0].labels.length){return this.pushStack(this[0].labels)}
+var escapeSelector=$.ui.escapeSelector=(function(){var selectorEscape=/([!"#$%&'()*+,./:;<=>?@[\]^`{|}~])/g;return function(selector){return selector.replace(selectorEscape,"\\$1")}})();/*!*jQuery UI Labels 1.12.1*http://jqueryui.com**Copyright jQuery Foundation and other contributors*Released under the MIT license.*http://jquery.org/license*/var labels=$.fn.labels=function(){var ancestor,selector,id,labels,ancestors;if(this[0].labels&&this[0].labels.length){return this.pushStack(this[0].labels)}
```

Comment:
Same note here. Also potentially troublesome because this removes newlines, which could be needed for ASI